generator/src/routes/+page.svelte

```diff
@@ -1,59 +1,246 @@
+  import logoTagsSvg from "$assets/ioxio-tags-logo.svg?url"
+  import effectSvg from "$assets/effect.svg?url"
+  import logomarkSvg from "$assets/ioxio-logomark.svg?url"
+  import logoSvg from "$assets/ioxio-logo.svg?url"
+  import QuestionSvg from "$assets/question.svg?component"
+  import subtractSvg from "$assets/subtract.png"
+  import type { PageData } from "./$types"
+
+  export let data: PageData
+
+  let productOption: string
+  let signOption: string
 </script>
 
 <svelte:head>
-  <title>Home</title>
-  <meta name="description" content="Svelte demo app" />
+  <title>Generator</title>
+  <meta name="description" content="QRcode Generator" />
 </svelte:head>
 
-<section>
-  <h1>
-    <span class="welcome">
-      <picture>
-        <source srcset={welcome} type="image/webp" />
-        <img src={welcome_fallback} alt="Welcome" />
-      </picture>
-    </span>
-
-    to your new<br />SvelteKit app
-  </h1>
-
-  <h2>
-    try editing <strong>src/routes/+page.svelte</strong>
-  </h2>
+<div class="container">
+  <!-- Left Panel -->
+  <div class="form-wrapper">
+    <div class="title">Generate a product passport</div>
+    <img class="logomarkSvg" src={logomarkSvg} alt="" />
+    <form>
+      <div class="row">
+        <FormInputGroup name="domain" label="Issuer domain" placeholder="ex.tags.ioxio.dev" />
+      </div>
+      <div class="row">
+        <div class="toggle-row">
+          <Toggle options={["arbitrary", "premade"]} bind:value={productOption} />
+        </div>
+        <FormSelectGroup
+          name="product"
+          label="Product"
+          placeholder="Type a product"
+          options={data.options}
+        />
+      </div>
+      <div class="row">
+        <FormInputGroup name="productId" label="Product ID" placeholder="ex. VV123456-12" />
+        <div class="toggle-row">
+          <Toggle options={["signed", "unsigned"]} bind:value={signOption} />
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <FormCheckbox name="valid" label="Create valid signature" />
+          <Tooltip tip="Whats this?" top>
+            <span class="question-icon">
+              <QuestionSvg />
+            </span>
+          </Tooltip>
+        </div>
+      </div>
+      <div class="actions-wrapper">
+        <Button title="Generate IOXIO Tag" onClick={() => {}} />
+      </div>
+    </form>
+  </div>
+  <!-- Right Panel -->
+  <div class="qrcode-area-wrapper">
+    <div class="qrcode-area">
+      <div class="qrcode">
+        <img class="frame" src={subtractSvg} alt="" />
+        <img class="effect" src={effectSvg} alt="" />
+        <img class="logo" src={logoTagsSvg} alt="" />
+      </div>
+    </div>
+    <div class="status">IOXIO Tag generator</div>
+    <div class="product-area">
+      <div class="description">
+        <p>
+          This application serves as a demonstration platform for generating QR codes specifically
+          designed for Data Product Passports.
```

Comment:
```suggestion
          designed for Digital Product Passports.
```
Sorry about this, the text has been worked on a bunch and it will likely change more in the future too, this is just an important correction at this very moment :smile: